README.md

```diff
@@ -12,132 +12,19 @@ Packages for creating MyST websites themes using React and Remix.
 All dependencies for `myst-theme` are included in this repository (a monorepo!).
 The core themes are also included in this repository to aid in development.
 
-## What's inside?
+## Local development and live server preview
```

Comment:
Maybe just link to the docs, instead of replicating the TOC here, otherwise we have to keep in sync.

Reply:
Could GH handle some include instead instead of just linking? 

I envision it could be useful for those who read the readme on the repo. But otherwise I agree don't copy paste any content as it will just get outdated really quickly